### PR TITLE
test: migrations file

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
This is an implementation to test and avoid future migration issues.

This test covers two cases
* The latest version of migration is updated with the number of migrations
* The latest migration implemented it's returning the state

**Issue**

Progresses #6818 

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by Gudahtt — DISMISSED** [review]
LGTM!

**Comment by tommasini** [comment]
Yeah, you are right! If we change the state we are actually breaking the test!

It's actually a good idea to have each member write a test if a migration it's implemented!!

Will push that code asap

cc: @Gudahtt What do you think about this approach?

**Review by MarioAslau — DISMISSED** [review]
Looking good!

**Review by Gudahtt — DISMISSED** [review]
Makes sense!

File: app/store/migrations.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { migrations, version } from './migrations';
+
+describe('Redux Persist Migrations', () => {
+  it('should apply last migration version and return state', () => {
+    let currentState = {
```
Comment (Cal-L) [inline app/store/migrations.test.js:5]: rename to oldState

File: app/store/migrations.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      recents: '0x1',
+    };
+
+    const migration = migrations[version];
+
+    currentState = migration(currentState);
```
Comment (Cal-L) [inline app/store/migrations.test.js:11]: rename to newState

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**